lib/evaluate/intrinsics.cc

```diff
@@ -1432,7 +1422,7 @@ std::optional<SpecificCall> IntrinsicProcTable::Implementation::Probe(
         bool ok{false};
```

Comment:
Would it make sense to make the semantics checking of "PRESENT" a function? It seems like it may need to be checked in more than one place.

Reply:
How would that happen?